boards/DIM/Src/Io/Io_Adc.c

```diff
@@ -0,0 +1,28 @@
+#include <stm32f3xx.h>
+#include "Io_SharedAdc.h"
+#include "Io_Adc.h"
+
+enum
+{
+    CHANNEL_12,
```

Comment:
Should this be called REGEN_PADDLE or something more descriptive? I just called it by the channel number in the previous PR because I was to lazy to check what that channel was used for.

Reply:
I don't think this is the right place to include a descriptive name because this file is just related to ADC itself. It should have no knowledge about what other modules are using it.

Reply:
👍Sounds good

Reply:
Where does the `CHANNEL_12` name come from? I don't see any reason that we wouldn't want a more descriptive name here? I totally agree that we don't want to add _extra_ context to the ADC, but since we need a name here anyways we might as well use a descriptive one?

Reply:
When we specify a channel to a rank in cube we have to specify the channel by its number and not its name. I think channel 12 is okay , imo

Reply:
Some good concerns were brought up and I added a big block comment clarifying key ADC concepts. Can everyone review it and let me know what you think: bc5a4866db2c4165b430da0e02325ab29763b474. It's meant for someone with limited knowledge about the technical details specific to STM32.

Reply:
This seems good to me now, thanks for clarifying.